Firmware/ultralcd.cpp

```diff
@@ -542,10 +542,14 @@ void lcdui_print_status_line(void) {
             break;
         }
     }
-    else if ((IS_SD_PRINTING) &&
+    else if (((IS_SD_PRINTING) &&
         (custom_message_type == CustomMsg::Status) &&
         (lcd_status_message_level <= LCD_STATUS_INFO) &&
         lcd_status_message_timeout.expired_cont(LCD_STATUS_INFO_TIMEOUT))
+#ifdef SHOW_FILENAME_AFTER_FINISH
+        || (GetPrinterState() == PrinterState::SDPrintingFinished)
```

Comment:
Minor note. This will overwrite any critical error messages such as from the thermal model.

Reply:
@gudnimg please check again. Fixed and tested https://github.com/prusa3d/Prusa-Firmware/pull/4821/commits/e427c4fb4e2f9ad3c63b8f5d22010e8fb182cf7b

Reply:
Looks good now 👍